Assets/Scripts/GameManager.cs

```diff
@@ -10,6 +10,7 @@ public class GameManager : MonoBehaviour {
     public GameObject bulletPowerupPrefab;
     public GameObject shipControlPowerupPrefab;
     public GameObject doubleShotPowerupPrefab;
+	public GameObject addLifePowerupPrefab;
```

Comment:
```c#
public GameObject addLifePowerupPrefab;
```

Make this line not indented or spaced or whatever it is, I cannot edit your edits to the document lol, but I can comment.